download.py

```diff
@@ -92,50 +134,70 @@ def get_zone_links(czds_base_url):
     exit(1)
 
 
+##############################################################################################################
+# Fourth Step: Set up AWS integration
+##############################################################################################################
+
+boto3.setup_default_session(**aws_session_args)
+aws_cloudwatch = boto3.client('cloudwatch', region_name=aws_region)
+
 
 ##############################################################################################################
-# Fourth Step: download zone files
+# Fifth Step: download zone files
 ##############################################################################################################
 
 # Function definition to download one zone file
 def download_one_zone(url, output_directory):
     print("{0}: Downloading zone file from {1}".format(str(datetime.datetime.now()), url))
 
     global  access_token
```

Comment:
I think at least nothing really changed from here to `193` ; just the `try` `except` was added so indenting changed.

Reply:
Yeah, I just added the try/catch, the HTTP status > 400 logic, and raised the exception to keep execution of the for loop which is ensuring the exception results in a CloudWatch metric update.